Sources/StreamChatUI/Components.swift

```diff
@@ -78,9 +78,9 @@ public struct Components {
     
     /// Object responsible for providing resizing operations for `UIImage`
     public var imageProcessor: ImageProcessor = NukeImageProcessor()
-
-    /// The object that loads previews for video attachments.
-    public var videoPreviewLoader: VideoPreviewLoader = DefaultVideoPreviewLoader()
+    
+    /// The object that loads previews for video attachments and resloves assets for a video player.
```

Comment:
I think this is more succinct and enough 👍 
```suggestion
    /// The object responsible for loading video attachments.
```

Reply:
✅